src/CoreGraphics/CGPDFScanner.cs

```diff
@@ -230,5 +230,30 @@ public bool TryPop (out CGPDFStream? value)
 				return false;
 			}
 		}
+
+#if NET
+		[SupportedOSPlatform ("ios16.0")]
+		[SupportedOSPlatform ("maccatalyst16.0")]
+		[SupportedOSPlatform ("macos13.0")]
+		[SupportedOSPlatform ("tvos16.0")]
+#else
+		[Mac (13,0), iOS (16,0), TV (16,0), MacCatalyst (16,0)]
+#endif
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		[return: MarshalAs (UnmanagedType.I1)]
```

Comment:
This method doesn't return a value, so this attribute seems unnecessary.

Reply:
Copy paste error.

Reply:
Fixed.